### PR TITLE
Prevent link focus styles from affecting the buttons block (Dark mode)

### PR DESCRIPTION
**Pull request by kjellr**
Same as #910, but for dark mode.

### CODE REVIEW

**Comment by carolinan** [comment]
Ah, there is one link text that has the wrong color:
image

**Comment by kjellr** [comment]
I think that's unrelated to this PR. I'm seeing these in `trunk` too:

Screen Shot 2020-12-01 at 11 51 51 AM
 too: 

Screen Shot 2020-12-01 at 11 50 09 AM

**Comment by carolinan** [comment]
it is because of the :not style added to the link focus style

**Comment by carolinan** [comment]
I mean for the meta nav

File: assets/sass/style-dark-mode.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		--button--color-background-active: var(--global--color-background);
 		--global--color-border: #9ea1a7;
 
-		.site a:focus,
+		.site a:focus:not(.wp-block-button__link):not(.wp-block-file__button),
 		.site a:focus .meta-nav {
```
Comment (carolinan) [inline assets/sass/style-dark-mode.scss]:
```suggestion
		.site a:focus:not(.wp-block-button__link):not(.wp-block-file__button) .meta-nav {
```

Comment (carolinan) [reply assets/sass/style-dark-mode.scss]: This feels very hacky but this is the best I can come up with because of stress.

**Pull request merged**